yoyodyne/indexes.py

```diff
@@ -1,7 +1,7 @@
 """Indexes for symbols."""
 
-import pickle
 import os
+import pickle
```

Comment:
please revert

Reply:
line 65 `dictionary = pickle.load(source)` line 77 `pickle.dump(vars(self), sink)` ?

Reply:
No, lines 2, and 4. You changed the order of imports (pickle was already being imported) to be non-lexicographic.

Reply:
oh gotcha, isort was overzealous